packages/backend/src/peripherals/database/shared/BaseRepository.ts

```diff
@@ -49,7 +49,7 @@ export class BaseRepository {
     method: AddMethod<T, R>,
   ): AddMethod<T, R> {
     return this.wrap(method, (id) =>
-      this.logger.debug({ method: method.name, id: id.valueOf() }),
+      this.logger.info({ method: method.name, id: id.valueOf() }),
```

Comment:
Using info will produce too many logs for papertrail, so it's actually not useful because we'll get cut off early